lib/ts/nextjs.ts

```diff
@@ -127,6 +128,141 @@ export default class NextJS {
             });
         };
     }
+
+    private static async commonSSRSession(
+        baseRequest: PreParsedRequest,
+        options?: VerifySessionOptions
+    ): Promise<{
+        session: SessionContainer | undefined;
+        hasToken: boolean;
+        hasInvalidClaims: boolean;
+        baseResponse: CollectingResponse;
+        nextResponse?: Response;
+    }> {
+        let baseResponse = new CollectingResponse();
+
+        try {
+            let session = await Session.getSession(baseRequest, baseResponse, options);
```

Comment:
this takes in a user context. So make that as in input to commonSSRSession as well, and likewise to all functions that call commonSSRSession

Reply:
done